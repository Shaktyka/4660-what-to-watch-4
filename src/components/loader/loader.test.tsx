import * as React from 'react';
import * as renderer from 'react-test-renderer';

import Loader from './loader';

describe(`Loader rendering`, () => {

  it(`Loader renders correctly`, () => {
    const tree = renderer
      .create(<Loader/>)
      .toJSON();

    expect(tree).toMatchSnapshot();
  });

});
