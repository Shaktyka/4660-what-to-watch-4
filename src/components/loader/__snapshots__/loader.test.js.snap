// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loader rendering Loader renders correctly 1`] = `
<div>
  <span
    style={
      Object {
        "color": "brown",
      }
    }
  >
    Loading ...
  </span>
</div>
`;
